File: D2L1/IncredibleMachineEasy.java
public class IncredibleMachineEasy {
    public double gravitationalAcceleration(int[] height,
                                            int T) {
        double sum = 0;
        for (int i = 0; i < height.length; ++i) {
            sum += Math.sqrt(2 * height[i]);
        }
        double ret = sum / T;
        return ret * ret;
    }
}

File: D2L1/IncredibleMachineEasy.cpp
#include <vector>
#include <string>
using namespace std;

double gravitationalAcceleration(vector<int> const &height,
                                 int T) {
    double sum = 0;
    for (int i = 0; i != height.size(); ++i)
        sum += sqrt(height[i]);
    double t = T/sum; 
    return 2/(t*t);
}
